subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/data/ci-artifacts.plots

```diff
@@ -1,13 +1,4 @@
-report: Error report
```

Comment:
should be removed

Reply:
I think you did it? I squashed all commits earlier.